plugins/core/IO/qLASIO/src/LasOpenDialog.cpp

```diff
@@ -401,3 +409,21 @@ void LasOpenDialog::onCurrentTabChanged(int index)
 		applyAllButton->setText(APPLY_ALL_TEXT);
 	}
 }
+
+void LasOpenDialog::onDecomposeClassificationToggled(bool checked) {
+	static constexpr std::array<const char *, 3>flagNames = {
+			LasNames::SyntheticFlag, LasNames::KeypointFlag, LasNames::WithheldFlag};
+
+	for (const char *flagName: flagNames) {
```

Comment:
Actually, it's as if the clang format was not applied?

Reply:
No i forgot to run it

Potentially we could add formatting checks in the CI

It would also be cool to have a clang format for the whole project

Reply:
Yes, we should apply it to the whole project and make a huge commit